src/input/tests/interactions/good/mie.toml

```diff
@@ -0,0 +1,58 @@
+[input]
+version = 1
+
+[global]
+cutoff = "3 A"
+tail_correction = true
+
+[[pairs]]
+atoms = ["A", "B"]
+mie = {sigma = "3 A", epsilon = "5.9 kJ/mol", n = 12.0, m = 6.0}
```

Comment:
:+1: on adding a separate file for this, could you remove the rest of the file here ? It is not really needed.